gov2/lambda/CreateFunction/CreateFunction.go

```diff
@@ -0,0 +1,117 @@
+//Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.\
+//SPDX-License-Identifier: Apache-2.0
+
+package main
+
+import (
+	"context"
+	"log"
+	"os"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/config"
+	"github.com/aws/aws-sdk-go-v2/credentials"
+	"github.com/aws/aws-sdk-go-v2/service/lambda"
+	"github.com/aws/aws-sdk-go-v2/service/lambda/types"
+)
+
+//This example creates a Lambda Function in your AWS Account using the Go V2 SDK. Note that it requies you
+// to have a zip file of the actual code already created and uploaded to an S3 Bucket.
+func main() {
+
+	//Your unique AWS Account Id
+	awsAccountId := os.Getenv("AWS_ACCOUNT_ID")
+
+	//example role that can be attached to the function. Here, we are using a role for dynamodb
+	exampleRole := "lambda-dynamodb-role"
+
+	//a unique function name. Your code can decide this
+	lambdaFunctionName := "Test_Function_Name_xcdff3453"
+
+	handlerName := "index.handler"
+	roleName := "arn:aws:iam::" + awsAccountId + ":role/" + exampleRole
```

Comment:
In the absence of the lambda handler code, I find this a little confusing to construct a role ARN and pass it in without any context.